engine/src/main/java/io/camunda/zeebe/engine/processing/streamprocessor/StreamProcessor.java

```diff
@@ -288,17 +288,21 @@ private ZeebeDbState recoverState() {
     return zeebeState;
   }
 
-  private void onRecovered(final long lastReprocessedPosition) {
+  private void onRecovered(final long lastSourceEventPosition) {
     phase = Phase.PROCESSING;
 
     // enable writing records to the stream
     processingContext.enableLogStreamWriter();
 
+    // Replay ends at the end of the log and returns the lastSourceEventPosition
+    // we need to seek to the next record after that position where the processing should start
+    // Be aware on processing we ignore events, so we will process the next command
+    logStreamReader.seekToNextEvent(lastSourceEventPosition);
```

Comment:
:thought_balloon: What do you think about moving the seek call into `startProcessing(lastSourceEventPosition)`? 

So, seeking to the start position is the first thing in the state machine when the processing starts. This would make it even more clear where the processing starts.